### DISCUSSION TITLE
Testing Translation

### DISCUSSION THREAD

**Discussion by avma** [pawelsalawa/sqlitestudio#3463] [Q&A]
Hi, 
Is it possible to test the translation i'm working on before submitting it?
Thanks 
Avi

**Comment by pawelsalawa** [comment]
Yes, you can. I don't know what happened with instructions for this, but I could not find them on the wiki, so I created a fresh instructions here:

https://github.com/pawelsalawa/sqlitestudio/wiki/Language_translations#testing-your-translations

**Comment by avma** [comment]
Thanks...
